src/components/remote-process.ts

```diff
@@ -0,0 +1,150 @@
+import { customElement } from "lit/decorators.js";
+import { StreamError, streamLogs } from "../api";
+import { ColoredConsole } from "../util/console-color";
+import { fireEvent } from "../util/fire-event";
+
+@customElement("esphome-remote-process")
+class ESPHomeRemoteProcess extends HTMLElement {
+  public type!: "validate" | "logs" | "upload";
+  public filename!: string;
+
+  private _abortController?: AbortController;
+  private _setup = false;
+
+  public connectedCallback() {
+    if (this._setup) {
```

Comment:
Shouldn't we restart what we aborted if we connect again?

Reply:
Nah we should create a new process